### PR TITLE
fix uppercase bug

### PR DESCRIPTION
**Pull request by ank27**
PRs must be submitted under the terms of our Contributor License Agreement [CLA](https://github.com/mapbox/mapbox-maps-android/blob/main/CONTRIBUTING.md#contributor-license-agreement).
Fixes: < Link to related issues that will be fixed by this pull request, if they exist >

## Pull request checklist:
 - [ ] Briefly describe the changes in this PR.
 - [ ] Include before/after visuals or gifs if this PR includes visual changes.
 - [ ] Write tests for all new functionality. If tests were not written, please explain why.
 - [ ] Add example if relevant.
 - [ ] Document any changes to public APIs.
 - [ ] Apply changelog label ('breaking change', 'bug :beetle:', 'build', 'docs', 'feature :green_apple:', 'performance :zap:', 'testing :100:') or use the label 'skip changelog'
 - [ ] Add an entry inside this element for inclusion in the `mapbox-maps-android` changelog: `<changelog></changelog>`.

### Summary of changes
Fix uppercase bug.

### CODE REVIEW

File: plugin-annotation/src/main/java/com/mapbox/maps/plugin/annotation/generated/PointAnnotation.kt [hunk 1]
```diff
@@ -84,7 +84,7 @@ class PointAnnotation(
     get() {
       val value = jsonObject.get(PointAnnotationOptions.PROPERTY_ICON_ANCHOR)
       value?.let {
-        return IconAnchor.valueOf(it.asString.uppercase())
+        return IconAnchor.valueOf(it.asString.toUpperCase(Locale.US))
```
Comment (kiryldz) [inline plugin-annotation/src/main/java/com/mapbox/maps/plugin/annotation/generated/PointAnnotation.kt:87]: Why `US`? 😄 
Guess `Locale.getDefault()` should fit better

**Review by kiryldz — COMMENTED** [review]
As this bug was introduced only recently label should be `skip changelog`.

File: plugin-annotation/src/main/java/com/mapbox/maps/plugin/annotation/generated/PointAnnotation.kt (hunk 1 above)
Comment (tobrun) [reply plugin-annotation/src/main/java/com/mapbox/maps/plugin/annotation/generated/PointAnnotation.kt:87]: actually specific US locale might be better. Have seen issues in the past with not using a specific locale. 
Will try finding that issue in old the repo.

**Comment by ank27** [comment]
changed the label and fix extra line after import.

File: plugin-annotation/src/main/java/com/mapbox/maps/plugin/annotation/generated/PointAnnotation.kt (hunk 1 above)
Comment (kiryldz) [reply plugin-annotation/src/main/java/com/mapbox/maps/plugin/annotation/generated/PointAnnotation.kt:88]: tbh in this current scenario almost sure it does not matter, but if default locale may cause issues, I'm totally fine with `US`.

**Review by kiryldz — APPROVED** [review]
Don't forget to update internal repo as well

File: plugin-annotation/src/main/java/com/mapbox/maps/plugin/annotation/generated/PointAnnotation.kt (hunk 1 above)
Comment (tobrun) [reply plugin-annotation/src/main/java/com/mapbox/maps/plugin/annotation/generated/PointAnnotation.kt:88]: https://developer.android.com/reference/java/util/Locale.html 
> Note that there are many convenience methods that automatically use the default locale, but using them may lead to subtle bugs.

**Review by pengdev — APPROVED** [review]

File: plugin-annotation/src/main/java/com/mapbox/maps/plugin/annotation/generated/PointAnnotation.kt (hunk 1 above)
Comment (ank27) [reply plugin-annotation/src/main/java/com/mapbox/maps/plugin/annotation/generated/PointAnnotation.kt:88]: I saw we have used `Locale.US` in other generated files as well, so used the same for now. but yeah we can change it with default locale everywhere in one go.

Comment (kiryldz) [reply plugin-annotation/src/main/java/com/mapbox/maps/plugin/annotation/generated/PointAnnotation.kt:88]: No need, let's stick to `US` as @tobrun mentioned above that using default may lead to bugs.

**Pull request merged**